doc/release-notes/8611-DataCommons-related-notes.md

```diff
@@ -0,0 +1,78 @@
+- Archiving via RDA-conformant Bags is more robust and is more configurable (PR #8773, 8747, 8699, 8609, 8606, 8610)
+- Users and administrators can see the archival status of the versions of the datasets they manage in the dataset page version table (PR #8748, #8696)
+- Administrators can configure messaging between Dataverse and other repositories that may hold related resources or services intersted in activity within Dataverse (PR #8775)
+
+## Notes for Dataverse Installation Administrators
+
+### Enabling experimental capabilities
+
+Several of the capabilities introduced in v5.12 are 'experimental' in the sense that further changes and enhancements to these capabilities should be expected and that these changes may involve additional work, for those who use the initial implementations, when upgrading to newer versions of Dataverse. Administrators wishing to use them are encouraged to stay in touch, e.g. via the Dataverse Community Slack space, to understand the limits of current capabilties and to plan for future upgrades.
+
+## New JVM Options and DB Settings
+
+The following DB settings have been added:
+
+- `:LDNMessageHosts`
+- `: BasicGlobusToken`
+- `:GlobusEndpoint`
+- `:GlobusStores`
+- `:GlobusAppUrl`
+- `:S3ArchiverConfig`
+- `:S3ArchiverProfile`
+- `:DRSArchivalConfig`
+
+See the [Database Settings](https://guides.dataverse.org/en/5.11/installation/config.html#database-settings) section of the Guides for more information.
+
+## Notes for Developers and Integrators
+
+See the "Backward Incompatibilities" section below.
+
+## Backward Incompatibilities
+
+### OAI-ORE and Archival Bag Changes
+
+Earlier versions of the archival bags included the ingested (tab-separated-value) version of tabular files while providing the checksum of the original file (Issue #8849). This release fixes that by including the original file and its metadata in the archival bag. This means that archival bags created prior to this version do not include a way to validate ingested files. Further, it is likely that capabilities in development (i.e. as part of the (Dataverse Uplaoder)[https://github/org/GlobalDataverseCommunityConsortium/dataverse-uploader]) to allow re-creation of a dataset version from an archival bag will only be fully compatible with archival bags generated by a Dataverse instance at a release > v5.12. (Specifically, at a minimum, since only the ingested file is included in earlier archival bags, an upload via DVUploader would not result in the same original file/ingested version as in the original dataset.) Administrators should be aware that re-creating archival bags, i.e. via the new batch archiving API, may be advisable now and will be recommended at some point in the future (i.e. there will be a point where we will start versioning archival bags and will start maintaining backward compatibility for older versions as part of transitioning this from being an experimental capability).
```

Comment:
```suggestion
Earlier versions of the archival bags included the ingested (tab-separated-value) version of tabular files while providing the checksum of the original file (Issue #8449). This release fixes that by including the original file and its metadata in the archival bag. This means that archival bags created prior to this version do not include a way to validate ingested files. Further, it is likely that capabilities in development (i.e. as part of the [Dataverse Uploader](https://github/org/GlobalDataverseCommunityConsortium/dataverse-uploader) to allow re-creation of a dataset version from an archival bag will only be fully compatible with archival bags generated by a Dataverse instance at a release > v5.12. (Specifically, at a minimum, since only the ingested file is included in earlier archival bags, an upload via DVUploader would not result in the same original file/ingested version as in the original dataset.) Administrators should be aware that re-creating archival bags, i.e. via the new batch archiving API, may be advisable now and will be recommended at some point in the future (i.e. there will be a point where we will start versioning archival bags and will start maintaining backward compatibility for older versions as part of transitioning this from being an experimental capability).
```